MyPy found new errors

https://github.com/just-work/fffw/actions/runs/5808758985/job/15746207725

<details><summary>Details</summary>
<p>

```
Run mypy --config-file=mypy.ini fffw
fffw/wrapper/params.py:[7](https://github.com/just-work/fffw/actions/runs/5808758985/job/15746207725#step:5:8): error: Incompatible default for argument "render" (default has type "None", argument has type "Callable[[Any], Any]")  [assignment]
fffw/wrapper/params.py:7: note: PEP 4[8](https://github.com/just-work/fffw/actions/runs/5808758985/job/15746207725#step:5:9)4 prohibits implicit Optional. Accordingly, mypy has changed its default to no_implicit_optional=True
fffw/wrapper/params.py:7: note: Use https://github.com/hauntsaninja/no_implicit_optional to automatically upgrade your codebase
fffw/graph/base.py:81: error: Incompatible default for argument "dst" (default has type "None", argument has type "Dest | Node")  [assignment]
fffw/graph/base.py:81: note: PEP 484 prohibits implicit Optional. Accordingly, mypy has changed its default to no_implicit_optional=True
fffw/graph/base.py:81: note: Use https://github.com/hauntsaninja/no_implicit_optional to automatically upgrade your codebase
Found 2 errors in 2 files (checked 20 source files)
Error: Process completed with exit code 1.

</p>
</details>

--- a/fffw/graph/base.py
+++ b/fffw/graph/base.py
@@ -78,7 +78,7 @@ class Dest(Traversable):
     def edge(self) -> Optional["Edge"]:
         return self._edge
 
-    def get_meta_data(self, dst: OutputType = None) -> Optional[Meta]:
+    def get_meta_data(self, dst: Optional[OutputType] = None) -> Optional[Meta]:
         if self._edge is None:
             raise RuntimeError("Dest not connected")
         return self._edge.get_meta_data(self)

--- a/fffw/wrapper/params.py
+++ b/fffw/wrapper/params.py
@@ -4,7 +4,7 @@ from typing import Any, Optional, Tuple, cast, List, Callable
 
 def param(default: Any = None, name: Optional[str] = None,
           stream_suffix: bool = False, init: bool = True, skip: bool = False,
-          render: Callable[[Any], Any] = None
+          render: Optional[Callable[[Any], Any]] = None
           ) -> Any:
     """
     Command line and filter parameters constructor.

